src/cli/utils.py

```diff
@@ -13,14 +15,101 @@ def read_json_file(file: Path, encoding: str = 'utf-8') -> Any:
+    """
+     Open the 'file' & write the 'data' to the 'file'
+
+    Args:
+        file: Path specifying the file to be written to
+        data: Data to be written to file
+        encoding: Encoding format in which to open the file
+    """
+
+    with open(file, mode="w", encoding=encoding) as fp:
+        fp.write(data)
+
+def get_network_nodes() -> Dict[str, List[str]]:
+    """
+    Loops trough all the networks in networks/networks.json and gets all the nodes within this network.
+
+    returns:
+    Returns a dict of networks, each having their own list including the nodes:
+    E.g.: {"randstad": ["leiden", "amsterdam", "the_hague"], "the-netherlands": ["etc..",]}
+
+    """
+    networks_file = Path(BASE_DIR) / "networks/networks.json"
+    channels_file = Path(BASE_DIR) / "networks/channels.json"
+
+    # Read network and see which are available
+    network_nodes: Dict[str, List[str]] = {}
+    with open(networks_file, "r", encoding="utf-8") as networks_fp:
+        data_networks = json.load(networks_fp)
+        with open(channels_file, "r", encoding="utf-8") as channels_fp:
+            data_channels = json.load(channels_fp)
+            for network in data_networks["networks"]:
+                for channel in data_channels["channels"]:
+                    if channel["slug"] in data_networks["networks"][network]["channels"]:
+                        if data_networks["networks"][network]["slug"] not in network_nodes:
+                            network_nodes[data_networks["networks"][network]["slug"]] = []
+                        lst = network_nodes[data_networks["networks"][network]["slug"]]
+                        if channel["node1"] not in lst:
+                            lst.append(channel["node1"])
+                        if channel["node2"] not in lst:
+                            lst.append(channel["node2"])
+                        network_nodes[data_networks["networks"][network]["slug"]] = lst
+
+    return network_nodes
+
+
+def get_dummy_application() -> Dict[str, Any]:
+    dummy_application = {
+        "application": [
+            {
+                "title": "Title for this input parameter",
+                "slug": "slug_for_this_input",
+                "description": "Description of this input",
+                "values": [
+                    {
+                        "name": "x",
+                        "default_value": 0,
+                        "minimum_value": 0,
+                        "maximum_value": 1,
+                        "unit": "",
+                        "scale_value": 1.0
+                    }
+                ],
+                "input_type": "number",
+                "roles": [
+                    "role1_given_in_command"
+                ]
+            }
+        ]
+    }
+
+    return dummy_application
+
+
+def get_py_dummy() -> str:
+    dummy_main = 'def main():\n    # Put your code here\n    pass\n\n\nif __name__ == "__main__": \n    main()\n'
```

Comment:
This should return the string which will be used as the template for the role.py files. I don't think this should return an empty dict?

Reply:
Of course it should return the template code. I meant the return inside your template code (instead of pass was my comment). See suggestion.
```suggestion
    dummy_main = 'def main():\n    # Put your code here\n    return {}\n\n\nif __name__ == "__main__": \n    main()\n'
```